docs/cloud-providers.md

```diff
@@ -8,6 +8,21 @@ k0s builds Kubernetes components in *providerless* mode, meaning that cloud prov
 
 Even when all components are built with providerless mode, you must be able to enable cloud provider mode for kubelet. To do this, run the workers with `--enable-cloud-provider=true`.
 
+When deploying with [k0sctl](k0sctl-install.md), you can add this into the `installFlags` of each host.
+
+```yaml
+spec:
+  hosts:
+  - ssh:
+      address: 10.0.0.1
+      user: root
+      keyPath: ~/.ssh/id_rsa
+    installFlags:
+      - --enable-cloud-provider
+      - --kubelet-extra-args="--cloud-provider=external"
```

Comment:
Adding kubelet args for plain `controller` role node will probably make it fail start. At least it _slightly_ confusing :)

Reply:
That's a good point. I install with controller+worker configuration for controller nodes (to service stuff that wants/needs to run on controller nodes). Might be worth clarifying in the docs. I'll consider how to word this best.